dockstore-common/src/main/java/io/dockstore/common/yaml/Service12.java

```diff
@@ -23,7 +23,7 @@
  * And the subclass property is an enum, not a string.
  *
  */
-public class Service12 extends AbstractYamlService {
+public class Service12 extends AbstractYamlService implements Workflowish {
```

Comment:
Kinda wonder if there should be a root class for the various stuff in the `yaml` package

Reply:
All classes that extend `Service12` and `YamlWorkflow` will implement `Workflowish` by definition.  Other classes that represent a workflow-type object can implement `Workflowish` as needed.